### PR TITLE
Made simulator more like board. Made simulator sample rate be 250Hz. …

### PR DESCRIPTION
**Pull request by pushtheworldllc**
Fixed start, stop, start stream bug. Fixed bugs with simulator sample number not rolling over.

### CODE REVIEW

File: openBCIBoard.js [hunk 1]
```diff
@@ -472,6 +472,8 @@ function OpenBCIFactory() {
                 reject('Simulator Already Running');
             } else {
                 // start simulating
+                this.connected = true;
+                this.streaming = true;
                 this.isSimulating = true;
                 // generateSample is a func that takes the previous sample number
                 var generateSample = openBCISample.randomSample(this.numberOfChannels(),this.sampleRate());
```
Comment (andyh616) [inline openBCIBoard.js]: where is this.sampleRate coming from?  I can't seem to find it.  Also, can we make the sample rate a parameter that defaults to 250?  Austin was saying that 250 samples per second might overwhelm the browser, so having an option to simulate a slow rate would be useful as we start to implement the real-time plotting

Comment (pushtheworldllc) [inline openBCIBoard.js]: Andy, it comes from the diff on line 490... every 4ms this function fires!

File: openBCIBoard.js [hunk 2]
```diff
@@ -485,7 +487,7 @@ function OpenBCIFactory() {
                     this.emit('sample',newSample);
                     oldSample = newSample;
                     resolve();
-                }, 20);
+                }, 4);
```
Comment (pushtheworldllc) [inline openBCIBoard.js]: Andy look here!

Comment (andyh616) [inline openBCIBoard.js]: got it. so yea, next version, def make that a parameter!

**Comment by andyh616** [comment]
LGTM thanks @pushtheworldllc

**Pull request merged**